code/measures/nml_parseutil.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+import xml.etree.ElementTree as ET
+from dateutil import parser
+import sys
+
+class Article:
+    
+    def __init__(self, company, timestamp, headline, article_text):
```

Comment:
Are there more relevant variables for the sophisticated (non BOW measures) we will implement later?